### PR TITLE
Add jsoncpp/1.9.1

### PR DESCRIPTION
**Pull request by madebr**
Specify library name and version:  **jsoncpp/1.9.1**

Tested on Linux 32- and 64-bit

- [x] I've read the [guidelines](https://github.com/conan-io/conan-center-index/wiki#how-to-submit-a-pull-request) for contributing.
- [x] I've followed the [PEP8](https://www.python.org/dev/peps/pep-0008/) style guides for Python code in the recipes.
- [x] I've used the [latest](https://github.com/conan-io/conan/releases/latest) Conan client version.
- [x] I've tried at least one configuration locally with the
      [conan-center hook](https://github.com/conan-io/hooks.git) activated.

### CODE REVIEW

File: recipes/jsoncpp/all/conanfile.py [hunk 1]
```diff
@@ -0,0 +1,67 @@
...
+        cmake.configure()
+        cmake.build()
+
+    def package(self):
+        self.copy("LICENSE", src=self._source_subfolder, dst="licenses")
+
+        cmake = self._get_cmake()
+        cmake.install()
+    def package_info(self):
+        self.cpp_info.name = "JsonCPp"
```
Comment (danimtb) [inline recipes/jsoncpp/all/conanfile.py]: Not sure, but this name looks wrong
```suggestion
        self.cpp_info.name = "JsonCpp"
```

Comment (madebr) [reply recipes/jsoncpp/all/conanfile.py]: This looks very wrong indeed :smile:

File: recipes/jsoncpp/all/conanfile.py [hunk 2]
```diff
@@ -0,0 +1,67 @@
...
+        return cmake
+
+    def build(self):
+        self._patch_sources()
+        cmake = self._configure_cmake()
+        cmake.build()
+
+    def package(self):
+        self.copy("LICENSE", src=self._source_subfolder, dst="licenses")
+
```
Comment (danimtb) [inline recipes/jsoncpp/all/conanfile.py]:
```suggestion
```

File: recipes/jsoncpp/all/conanfile.py [hunk 3]
```diff
@@ -0,0 +1,67 @@
...
+        cmake = self._configure_cmake()
+        cmake.install()
```
Comment (danimtb) [inline recipes/jsoncpp/all/conanfile.py]:
```suggestion
        cmake.install()

```

**Comment by SSE4** [comment]
re-opening to trigger CLA

**Pull request merged**